src/MakePIAPortable/MakePIAPortable.cs

```diff
@@ -37,6 +37,47 @@ class Program
             { "System.ValueType", System_Runtime },
             { "System.Collections.Generic.IEnumerable", System_Runtime },
             { "System.Reflection.DefaultMemberAttribute", System_Runtime },
+            { "System.Reflection.AssemblyDelaySignAttribute", System_Runtime},
+            { "System.Runtime.CompilerServices.CompilationRelaxationsAttribute", System_Runtime },
+            { "System.Runtime.CompilerServices.RuntimeCompatibilityAttribute", System_Runtime },
+            { "System.Diagnostics.DebuggableAttribute", System_Runtime },
+            { "System.Reflection.AssemblyTitleAttribute", System_Runtime },
+            { "System.Runtime.Versioning.TargetFrameworkAttribute", System_Runtime },
+            { "System.Resources.SatelliteContractVersionAttribute", System_Runtime },
```

Comment:
This isn't in netstandard1.3 system.runtime.

Reply:
It should be in System.Resources.ResourceManager

Reply:
You could probably also just drop this attribute on the floor since we will never have a satellite dll.